.github/workflows/build-installers.yaml

```diff
@@ -258,32 +290,60 @@ jobs:
           echo "EXE_FILE=$EXE_FILE" >>$GITHUB_ENV
           echo "WEB_FILE=$WEB_FILE" >>$GITHUB_ENV
 
+      # Dev release should not be set as latest
+      - name: Decide if release should be set as latest
+        id: is_latest
+        shell: bash
+        run: |
+          unset IS_LATEST
+
+          echo "Github ref is $GITHUB_REF"
+
+          if [[ "$GITHUB_REF" == "refs/tags/dev" ]]; then
+            echo "dev tag matched"
+            IS_LATEST='false'
+            IS_PRERELEASE='true'
+          elif [[ "$GITHUB_REF" =~ "-rc" ]]; then
+            echo "release candidate tag matched"
+            IS_LATEST='false'
+            IS_PRERELEASE='true'
+          else
+            echo "main branch release matched"
+            IS_LATEST='true'
+            IS_PRERELEASE='false'
+          fi
+
+          echo "IS_LATEST=${IS_LATEST}" >> "$GITHUB_OUTPUT"
+          echo "IS_PRERELEASE=${IS_PRERELEASE}" >> "$GITHUB_OUTPUT"
+
       - name: Release
-        uses: softprops/action-gh-release@v0.1.15
+        uses: softprops/action-gh-release@v2.1.0
         with:
+          prerelease: ${{steps.is_latest.outputs.IS_PRERELEASE}}
+          make_latest: "${{steps.is_latest.outputs.IS_LATEST}}"
           files: |
             ${{ env.DMG_FILE }}
             ${{ env.DEB_FILE }}
             ${{ env.EXE_FILE }}
             ${{ env.WEB_FILE }}
 
       - name: Get repo name
+        if: startsWith(github.ref, 'refs/tags/') && !contains( github.ref, '-rc')
```

Comment:
Do you need to also exclude 'dev' here, or are you removing dev completely
If removing dev completely, can probably remove from the "is latest" logic above too?

Reply:
Good point - I was supporting both methods while I was working on this, but don't need the dev stuff anymore, I'll look at what I can remove related to that.

Reply:
Removed